application.conf

```diff
@@ -0,0 +1,12 @@
+ensime = {
+  server-conf = {
+    shutDownOnDisconnect = true
+    test = true
+    sys-protocol = "swank"
+    exitAfterIndex = true
+    disableSourceMonitoring = true
+    disableClassMonitoring = true
+    legacyJarUrls = true
+  }
+  xdg-conf-home = "~/.config/.ensime-server.conf"
```

Comment:
Leave `xdg-conf-home` as a directory and append filename later just for consistency.